superset/assets/src/dashboard/reducers/getInitialState.js

```diff
@@ -74,26 +74,28 @@ export default function(bootstrapData) {
   const sliceIds = new Set();
   dashboard.slices.forEach(slice => {
     const key = slice.slice_id;
-    chartQueries[key] = {
-      ...chart,
-      id: key,
-      form_data: slice.form_data,
-      formData: applyDefaultFormData(slice.form_data),
-    };
+    if (['separator', 'markup'].indexOf(slice.form_data.viz_type) === -1) {
+      chartQueries[key] = {
+        ...chart,
+        id: key,
+        form_data: slice.form_data,
+        formData: applyDefaultFormData(slice.form_data),
```

Comment:
I would <3 to rename `formData` to something more meaningful, I still don't understand what it is :(

Reply:
form_data is saved in table and passed from server-side. formData is after applied filter etc. This variable is generally used in dashboard and explore view...probably not a good place to rename from here.

Reply:
🙉 that is the worst naming ever! why not call it form_data_with_filters?! but yeah, sounds like we can't change it now. thanks for the deets.